File: c#/strings/SubstringSearch.cs
class SubstringSearch {
  public static void Main(string[] args) {
    string cite = "Stat rosa pristina nomine, nomina nuda tenemus.";
    System.Console.WriteLine(string.Format("cite = {0}\n", cite));

    int rosa_position = cite.IndexOf("rosa");
    System.Console.WriteLine(string.Format(
      "\"rosa\" position: {0} ({1})",
      rosa_position,
      rosa_position.GetType()
    ));

    int odonata_position = cite.IndexOf("odonata");
    System.Console.WriteLine(string.Format(
      "\"odonata\" position: {0} ({1})",
      odonata_position,
      odonata_position.GetType()
    ));
  }
}

File: php/strings/substring_search.php
<?php
/*
 * Copyright (C) 2019-2026 Katarzyna Krasińska
 * Languagium - https://github.com/katheroine/languagium
 * Licensed under GPL-3.0 - see LICENSE.md
 */

$cite = "Stat rosa pristina nomine, nomina nuda tenemus.";
print("cite = {$cite}\n\n");

$rosa_position = strpos($cite, "rosa");
print("\"rosa\" position: {$rosa_position} (" . gettype($rosa_position) . ")\n");

$odonata_position = strpos($cite, "odonata");
print("\"odonata\" position: {$odonata_position} (" . gettype($odonata_position) . ")\n");
